Dockerfile

```
@@ -190,6 +190,22 @@ RUN set -ex; \
         done; \
     fi
 
+# pgai is an extension for artificial intelligence
+ARG PGAI
```

All the other once have `_VERSION`, so for consistency, please:
```suggestion
ARG PGAI_VERSION
```

---

fixed